### PR TITLE
[G4] Update Geant4 to 11.1.ref07

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

**Comment by iarspider** [comment]
please test

**Comment by iarspider** [comment]
please test

File: geant4.spec [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+%define tag d35b0306a837c2b3640b250c1af599688eee8df1
 #%define branch cms/v%{realversion}
-#%define github_user cms-externals
-%define tag cf4c6712c60cdf2b4d8799e34e176917c2de358d
-%define branch master
-%define github_user Geant4
+%define github_user cms-externals
+#%define tag d35b0306a837c2b3640b250c1af599688eee8df1
+#%define branch master
+#%define github_user Geant4
```
Comment (smuzaffar) [inline geant4.spec]: please remove all these commented  out code ( this also break automatic forward ports)

**Comment by iarspider** [comment]
please test

**Comment by iarspider** [comment]
please test

**Comment by iarspider** [comment]
please test

**Pull request merged**